files/en-us/tools/working_with_iframes/index.html

```diff
@@ -10,14 +10,26 @@
 ---
 <div>{{ToolsSidebar}}</div>
 
-<p>You can point the developer tools at a specific <a href="/en-US/docs/Web/HTML/Element/iframe"><code>iframe</code></a> within a document.</p>
+<p>You can point the developer tools at a specific <a href="/en-US/docs/Web/HTML/Element/iframe"><code>iframe</code></a> within a document. The <a href="/en-US/docs/Tools/Page_Inspector">Inspector</a>, <a href="/en-US/docs/Tools/Web_Console">Console</a>, <a href="/en-US/docs/Tools/Debugger">Debugger</a> and all other developer tools will then target that iframe (essentially behaving as if the rest of the page does not exist).</p>
 
 <p>{{EmbedYouTube("Me9hjqd74m8")}}</p>
 
-<p>You'll see a button in the toolbar:</p>
+<p>To set an iframe as the target for the developer tools, first select the <em>iframe button</em> to launch a popup listing all the iframes in the document (and the main document itself). Then Select an entry to make it the <em>target iframe</em>. Note that the iframe button is not displayed if the page doesn't have any iframes!</p>
```

Comment:
```suggestion
<p>To set an iframe as the target for the developer tools, first select the <em>iframe button</em> to launch a popup listing all the iframes in the document (and the main document itself). Then select an entry to make it the <em>target iframe</em>. Note that the iframe button is not displayed if the page doesn't have any iframes!</p>
```
…assuming that *Select* was not capitalized intentionally; if it is in fact intentional, then please just dismiss this suggestion